tests/contest/contest/src/tests/process_capabilities_fail/process_capabilities_fail_test.rs

```diff
@@ -0,0 +1,79 @@
+use std::{collections::HashSet, fs, fs::OpenOptions, io::Write};
+
+use anyhow::{anyhow, Context, Ok, Result};
+use oci_spec::runtime::{Capability, LinuxCapabilitiesBuilder, ProcessBuilder, Spec, SpecBuilder};
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use serde_json::Value;
+
+use crate::utils::{test_inside_container, test_utils::CreateOptions};
+
+fn create_spec() -> Result<Spec> {
+    let linux_capability = LinuxCapabilitiesBuilder::default()
+        .bounding(HashSet::from([Capability::Syslog]))
+        .build()?;
+
+    let process = ProcessBuilder::default()
+        .args(vec![
+            "runtimetest".to_string(),
+            "process_capabilities_fail".to_string(),
```

Comment:
I don't think we need this here, the default command should be ok. We don't actually intend this to reach the runtimetest stage, but instead fail in the creation itself. Right now, if we add this, and somehow the container is created successfully ; then we would still get error from the runtime test, which would result in `test_inside_container` returning `Failed` and thus our test incorrectly passing. You can simply set the args to `sleep,1m` or such (check what we set by default in utils .